documentation/C++style.md

```diff
@@ -296,6 +297,23 @@ like `Http`.
 Consistency is one of many aspects in the pursuit of clarity,
 but not an end in itself.
 
+## <a name="checklist"></a>Pull request checklist
+Before submitting a pull request, review the following items:
+*  Verify that all files have a license.
+*  Make sure that all source lines have 80 or fewer characters.
+*  Run `git diff` on all files to look for spurious changes such as `include <iostream>`.
+*  Run clang-format version 7 on all .cc and .h files.
+*  Build and test with both GNU and clang compilers.
+*  Make sure that all error messages you added appear in tests.
+*  Annotate the code and tests with appropriate references to constraint and requirement numbers in the standard.
```

Comment:
This too.

Reply:
As above, I agree with you.  Should I remove it?

Reply:
+1

Reply:
Yes, thanks.